enterprise/frontend/src/metabase-enterprise/content_translation/components/ContentTranslationConfiguration.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  type ChangeEvent,
+  type ReactNode,
+  useCallback,
+  useRef,
+  useState,
+} from "react";
+import { t } from "ttag";
+
+import { UploadInput } from "metabase/components/upload";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import Markdown from "metabase/core/components/Markdown";
+import {
+  Form,
+  FormProvider,
+  FormSubmitButton,
+  useFormContext,
+} from "metabase/forms";
+import { useSelector } from "metabase/lib/redux";
+import { getDocsUrl } from "metabase/selectors/settings";
+import { Group, Icon, Loader, Stack, Text } from "metabase/ui";
+import { useUploadContentTranslationDictionaryMutation } from "metabase-enterprise/api";
+
+export const ContentTranslationConfiguration = () => {
+  const availableLocalesDocsUrl = useSelector((state) =>
+    // eslint-disable-next-line no-unconditional-metabase-links-render -- This is used in admin settings
+    getDocsUrl(state, {
+      page: "configuring-metabase/localization",
+      anchor: "supported-languages",
+    }),
+  );
```

Comment:
`useDocsUrl` makes this api a little nicer

```
const availableLocalesDocsUrl = useDocsUrl(
  "configuring-metabase/localization",
  { anchor: "supported-languages" }
);
```

Reply:
Cheers, this is nice